content/source/docs/cloud/api/state-version-outputs.html.md

```diff
@@ -14,6 +14,101 @@ State version outputs are the [output values](/docs/language/values/outputs.html
 the name and value of the output, as well as a sensitive boolean if the value
 should be hidden by default in UIs.
 
+## List State Version Outputs
+
+`GET /state-versions/:state_version_id/outputs`
+
+Listing state version outputs requires permission to read state outputs for the workspace. ([More about permissions.](/docs/cloud/users-teams-organizations/permissions.html))
+
+Parameter | Description
+----------|---------
+`:state_version_id` | The ID of the desired state version.
+
+Status  | Response                                     | Reason
+--------|----------------------------------------------|----------
+[200][] | [JSON API document][]                        | Successfully returned a list of outputs for the given state version
```

Comment:
should we add periods after the reasons? It feels like in some tables we do end with a period and in some we don't. I would add them for consistency, but let me know if it's the convention that we don't have them here for some reason :)

Reply:
Doing a quick visual scan of the docs, I see no period for the `Reason` column in the `Status Response` tables more frequently. Conversely, they are consistently used in `Request Body`, `Request Param` tables in the `Description` column. Would defer to @nfagerlund on this one.

Reply:
Ha, I totally missed that. 

If there's a convention here, I believe it's accidental! 

We should probably switch to using periods consistently in long table cells like these — either's fine in the case of a fragment, but when it might be either a fragment or a sentence, it's simpler to just put a dot. If you haven't merged it yet, we might as well start today, but it's clearly not urgent!

Reply:
Copy that, I'll add the periods and keep that in mind for any future docs. Thanks @laurapacilio and @nfagerlund !